cli-config.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
```

Comment:
Die Datei umgeht momentan unseren Coding Style Check. Sie sollte beschrieben werden und vermutlich im Test-Ordner liegen.

Reply:
Ich habe die Datei jetzt erst einmal explizit in den Check mit aufgenommen.